CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [1.0-beta] - 2016-09-14
+### Added
+- App Engine Standard support ([#767](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/767))
```

Comment:
A couple other changes that would be nice to include in the log:

https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/868 (exposing extra fields in the deployment config)
https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/811 (updating local run debug to use the newer tooling)